### ISSUE TITLE
3D source files

### ISSUE THREAD

**Issue by JonnyHaystack** [jfedor2/twist-to-scroll-trackball#1]
Hi, would you mind uploading the source files for the 3D models? At the moment I'm just trying to put a USB C port cutout in the wall instead of the hole for the cable, but it's a bit tricky to manually patch the cable hole closed. Alternatively, if you could just upload an STL for the bottom part without the cable hole that would make it a lot easier to edit it with a custom port cutout.

I was mostly able to do it with some hacky editing but it comes out a bit weird in the slicer and I'm not sure if it might cause problems when printing :-/
![image](https://user-images.githubusercontent.com/1266473/147754525-3107afc7-0aa2-47ac-9148-a418a294a944.png)

**Comment by jfedor2** [comment]
I can upload the source file, but for stuff like this you should really start with the STEP file, not the STLs. It won't give you my editing timeline, but removing the cable cutout is as easy as selecting three faces and pressing the del key (at least in Fusion 360).

**Comment by JonnyHaystack** [comment]
Thanks. I use OpenSCAD usually so I can't really do anything with the STEP file in that I don't think.

**Comment by jfedor2** [comment]
Yeah, the "source" I'm talking about would be a Fusion 360 file, so not useful in OpenSCAD either I'm afraid.

**Comment by JonnyHaystack** [comment]
Yeah I know, but in this case I would just remove the cable cutout in Fusion 360 and export it as an STL, then do my thing in OpenSCAD.

Oh, and on an unrelated note, I accidentally printed one of the mouse button things without supports, and it actually printed fine and came out better than the ones that I printed with supports. The first layer had a couple of droopy bits which I clipped off, but other than that it turns out great. My printer isn't anything crazy good either.
![image](https://user-images.githubusercontent.com/1266473/147840981-d1b29c7c-765d-4acd-b4a9-7aac40e0ef7f.png)
![image](https://user-images.githubusercontent.com/1266473/147840986-ee03f42d-8c9c-4181-bc38-a17fdd96d0db.png)
